.github/workflows/post-release.yml

```diff
@@ -89,6 +93,13 @@ jobs:
       with:
         toolchain: ${{ matrix.rust }}
         targets: ${{ matrix.target }}
+    - name: Set up Apple silicon support
+      if: matrix.target == 'aarch64-apple-darwin'
+      shell: bash
+      run: |
+        rustup target add aarch64-apple-darwin
```

Comment:
Won't the `rust-toolchain` step above take care of this?

Reply:
Sorry, I do not know.
I have zero knowledge of Rust environments.